app/models/article.rb

```diff
@@ -817,7 +817,7 @@ def validate_co_authors_exist
 
   def future_or_current_published_at
     # allow published_at in the future or within 15 minutes in the past
-    return if !published || published_at > Time.current - (15 * 60 * 60)
+    return if !published || published_at > Time.current - (15 * 60)
```

Comment:
**Suggestion:** Maybe we can use ActiveSupport's helper methods to make it more readable and less error prone 🙂 

```suggestion
    return if !published || published_at > 15.minutes.ago
```

Reply:
I don't know why I haven't used that :D